src/main/java/edu/ucsb/cs156/happiercows/jobs/UpdateCowHealthJob.java

```diff
@@ -46,22 +46,26 @@ public void accept(JobContext ctx) throws Exception {
                 User user = userRepository.findById(userCommons.getUserId()).orElseThrow(()->new RuntimeException("Error calling userRepository.findById(" + userCommons.getUserId() + ")"));
                 ctx.log("User: " + user.getFullName() + ", numCows: " + userCommons.getNumOfCows() + ", cowHealth: " + userCommons.getCowHealth());
 
-                double newCowHealth = calculateNewCowHealth(userCommons.getCowHealth(), userCommons.getNumOfCows(), totalCows, carryingCapacity, degradationRate);
-                ctx.log(" old cow health: " + userCommons.getCowHealth() + ", new cow health: " + newCowHealth);
-                userCommons.setCowHealth(newCowHealth);
+                if(userCommons.gameInProgress()){
+                    double newCowHealth = calculateNewCowHealth(userCommons.getCowHealth(), userCommons.getNumOfCows(), totalCows, carryingCapacity, degradationRate, userCommons.gameInProgress());
+                    ctx.log(" old cow health: " + userCommons.getCowHealth() + ", new cow health: " + newCowHealth);
+                    userCommons.setCowHealth(newCowHealth);
+                    ctx.log("Cow health has been updated!");
+                } else{
+                    ctx.log("Game is not in progress: cow health will not be updated for this commons.");
+                }
+                
                 userCommonsRepository.save(userCommons);
             }
         }
-
-        ctx.log("Cow health has been updated!");
     }
 
     public static double calculateNewCowHealth(
-            double oldCowHealth,
-            int numCows,
-            int totalCows,
-            int carryingCapacity,
-            double degradationRate) {
+        double oldCowHealth,
+        int numCows,
+        int totalCows,
+        int carryingCapacity,
+        double degradationRate, boolean gameInProgress) {
```

Comment:
Why do we need this?

We can just avoid calling the function in the first place.  Note that you didn't use the value inside the function.